src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -2433,6 +2574,57 @@ GNAPluginNS::ConnectionDetails GNAGraphCompiler::connectInput(CNNLayerPtr layer,
             gnamem->bind_ptr(layer, ptr, &cropLayerInfoItem.gna_ptr, offset);
             return CNNNetPrevLayer(prevLayer);
         }
+    } else if (layerInfoObj.isPad()) {
+        auto beforePad = CNNNetPrevLayerSkipCertain(prevLayer, 0, [](CNNLayerPtr l) {
+            return LayerInfo(l).isNonFunctional();
+            });
+
+        const auto padAligned = prevLayer->params.count("PadAligned") > 0 ||
```

Comment:
Could you please explain why we need too many special runtime marks? Let's imagine that model with a such subgraph came from the user. Why this model should not work? GNA should handle it without any marks.

Reply:
All RT info is already removed